packages/terra-show-hide/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ ChangeLog
 Unreleased
 ----------
 ### Changed
+* Update react/react-dom peer-dependency to latest stable minor release
+### Changed
```

Comment:
Double `Changed` sections here

If these changelog updates were done with a script it might be worth rerunning it and adding a newline to each section.

Reply:
Updated in these commits:
https://github.com/cerner/terra-core/pull/2298/commits/9be68d5c3c198a058b2c3b2d898aa74f23101e6b

https://github.com/cerner/terra-core/pull/2298/commits/b1f530a1e24083fc975e9f0b4b2a036add277dd0